src/proto/Algorand.proto

```diff
@@ -11,9 +11,25 @@ message TransactionPay {
     uint64 last_round = 5;
 }
 
+message AssetTransaction {
+    string to_address = 1;
+    uint64 fee = 2;
+    uint64 amount = 3;
+    uint64 first_round = 4;
+    uint64 last_round = 5;
+    uint64 asset_id = 6;
+}
+
+message AssetOptIn {
+    uint64 fee = 1;
+    uint64 first_round = 2;
+    uint64 last_round = 3;
```

Comment:
these two fields can be moved to `SigningInput`, same as `genesis_hash` needed by all transaction types

Reply:
It will break api of the library for people who used Algorand SigningInput before. I thought about it.

Reply:
It's ok, algo usage is low and just mentioned it's a breaking change